tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -82,6 +89,48 @@ def on_heartbeat(self, cluster_data):
             if data:
                 self.on_sync_object(data)
 
+        # Get and update ec profiles for the cluster
+        self._sync_ec_profiles()
+
+    def _sync_ec_profiles(self):
```

Comment:
This seems fine, we need to CLI when we dont have that data in maps

@shtripat We need comments wherever cli is used as to what is the expected input and output of the CLI, please add that in this PR